src/Illuminate/Auth/Authenticatable.php

```diff
@@ -25,6 +25,18 @@ public function getAuthPassword()
     }
 
     /**
+     * Automatically encrypt password if not done yet.
+     *
+     * @param $pass
+     */
+    public function setPasswordAttribute($value)
+    {
+        $value = app('hash')->needsRehash($value) ? bcrypt($value) : $value;
```

Comment:
In other words: both `User::create(['password' => 'myValue'])` as `User::create(['password' => bcrypt('myValue')])` will now insert the correct hashed value into the database.